modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -153,21 +172,66 @@ public static com.liferay.object.model.ObjectField toObjectField(
 			LocalizedMapUtil.getLocalizedMap(objectField.getLabel()));
 		serviceBuilderObjectField.setName(objectField.getName());
 		serviceBuilderObjectField.setObjectFieldSettings(
-			TransformUtil.transformToList(
-				objectField.getObjectFieldSettings(),
-				objectFieldSetting ->
-					ObjectFieldSettingUtil.toObjectFieldSetting(
-						objectField.getBusinessTypeAsString(),
-						objectFieldSetting, objectFieldSettingLocalService,
-						objectFilterLocalService)));
+			Collections.emptyList());
```

Comment:
As discussed in Slack. We add feature flag in `ObjectFieldSettingUtil:57`. So, if FF is true, we don´t set ObjectStateFlow. When remove the FF, we'll remove the if, because he is not necessary.